src/components/Toggle/Toggle.tsx

```diff
@@ -30,7 +33,9 @@ function Toggle(props: TogglesProps) {
         }`}
         data-testid="toggle-switch"
       >
-        {getColorTheme(props.isDarkThemeOff).toUpperCase()}
```

Comment:
Why did you delete your function? Keep using it please

Reply:
I can't use this fanction in this part.

Reply:
I made a mistake I can do it